CONVENTIONS.md

```diff
@@ -126,6 +126,20 @@ it is used to manage all elements of the cluster.
 * Every component must remain available to consumers without disruption during upgrades and reconfiguration
   * See the next section for more details
 
+#### High Availability
+
+We focus on minimizing the impact of single node failures by ensuring operators or operands are spread across multiple nodes while maintaining backwards compatibility. Currently, we mandate 3 control-plane nodes and 2 worker nodes in OpenShift, so the following scenarios account for components that are running on both control-plane and worker nodes. 
```

Comment:
Second sentence needs to describe compact clusters (which is equivalent to 3+3)

Reply:
It would be ideal if this new section was organized around the 3 supported deployment topologies (single-node, compact, and standard), with the conventions and expectations for each discussed separately.